visualisations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -0,0 +1,148 @@
+from webviz_plotly import Plotly
+import pandas as pd
+import matplotlib.cm as cm
+
+color_scheme = cm.get_cmap('Set1')
+
+
+def color_spread(n):
+    """Color generator function
+
+    :param data:
+        n = number of total colors needed
+        Returns list of lists of three strings representing rgb
+    """
+    colorlist = []
+    for i in range(n):
+        single_color = color_scheme(float(i) / n)
+        formatted_color = []
+        for y in single_color[:-1]:
+            formatted_color.append(str(y))
+        colorlist.append(formatted_color)
+    return colorlist
```

Comment:
1) Useful for many different visualizations. Should `color_spread` be lifted from `webviz_fan_chart` to `webviz`?

2) Considered using https://seaborn.pydata.org/tutorial/color_palettes.html#palette-tutorial instead?